jenkins/job-configs/kubernetes-jenkins-pull/kubernetes-pull.yaml

```diff
@@ -201,23 +201,6 @@
         - shell: JENKINS_BUILD_STARTED=true "${{WORKSPACE}}/_tmp/upload-to-gcs.sh"
         - shell: '{cmd}'
     publishers:
-        - xunit:
-            thresholds:
-                - failed:
-                    unstable: 0
```

Comment:
I wonder what happens if we unset this - will it make the plugin barf, or will it make the plugin parse test results but never fail?